project_resource_calendar/i18n/fr.po

```diff
@@ -729,9 +756,14 @@ msgid "Start Date"
 msgstr "Date de début"
 
 #. module: project_resource_calendar
-#: model:ir.model.fields,field_description:project_resource_calendar.field_doublebooking_validation_wiz_start_date
-msgid "Starting Date"
-msgstr "Date de début"
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:136
+#: code:addons/project_resource_calendar/static/src/js/save_event_wizard.js:23
+#: code:addons/project_resource_calendar/static/src/js/save_event_wizard.js:82
+#: code:addons/project_resource_calendar/static/src/js/save_event_wizard.js:86
+#, python-format
+msgid "Starting at"
+msgstr "Starting at"
```

Comment:
transaltion?

Reply:
I wasn't originally in the POT file I downloaded from Odoo to add new translations.